examples/29-vpc-with-ip-family.yaml

```diff
@@ -0,0 +1,14 @@
+# An example of ClusterConfig object with custom VPC ipFamily
+# set to ipv6; also without any nodegroups:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-2
+  region: eu-north-1
+
+vpc:
+  ipFamily: ipv6
+
+nodeGroups: []
```

Comment:
unmanaged nodegroups won't be supported so this line should be `managedNodeGroups`

Reply:
Ah, thanks! :D But does it even matter, if it's empty? 🤔 Can I even just delete it? :D

Reply:
it doesn't since it's empty! you could delete it :)